lib/checks/visibility/hidden-content.js

```diff
@@ -1,8 +1,15 @@
 let styles = window.getComputedStyle(node);
 
 if (axe.commons.dom.hasContent(node)) {
-  if (styles.getPropertyValue('display') === 'none' || styles.getPropertyValue('visibility') === 'hidden') {
+  if (styles.getPropertyValue('display') === 'none') {
     return undefined;
+  } else if (styles.getPropertyValue('visibility') === 'hidden') {
+    if (node.parentNode) {
+      var parentStyle = window.getComputedStyle(node.parentNode);
+    }
+    if (!parentStyle || parentStyle.getPropertyValue('visibility') !== 'hidden') {
+      return undefined;
```

Comment:
Ah, where to begin...

So, `display` is NOT inherited, however `visibility` IS inherited. We needed to add an additional layer here that checks child/parents to return appropriately. I worked with Wilco on this one earlier today.